admin/helpers/school-home-feature-eligibility-presenter.js

```diff
@@ -8,19 +8,32 @@ const dateService = require('../services/date.service')
 const schoolHomeFeatureEligibilityPresenter = {}
 
 /**
- * Determine the results eligibility date
+ * Determine the results earliest eligibility date when hdf has been submitted
```

Comment:
I think a Presenter should only be formatting data for display.  This is actual business logic that might be better in a service.

Also the description is unclear to me.  Perhaps 'Determine the earliest date the results page can be shown for schools that have submitted the HDF' is better.  It

Reply:
@jon-shipley I was thinking about this yesterday myself and for that reason I have extracted the result availability related logic pieces to a dedicated service.